### PR TITLE
TRM - Evpn multicast

### PR DESCRIPTION
**Pull request by rahushen**
Add node_utils logic for TRM (evpn multicast and changes to vrf_af to support mvpn).

### CODE REVIEW

File: tests/test_evpn_multicast.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/evpn_multicast'
+require_relative '../lib/cisco_node_utils/feature'
+
+# TestEvpnMulticast - Minitest for EvpnMulticast class
+class TestEvpnMulticast < CiscoTestCase
+  @skip_unless_supported = 'evpn_multicast'
+
+  def setup
```
Comment (mikewiebe) [inline tests/test_evpn_multicast.rb:27]: We have seen issues in subsequent tests if we don't include a `teardown` method.   Suggest you add one to remove `feature ngmvpn` and `advertise evpn multicast` when you are done.

Example:
https://github.com/cisco/cisco-network-node-utils/blob/develop/tests/test_acl.rb#L31

**Comment by mikewiebe** [comment]
@rahushen Minor test comment, otherwise 👍

**Comment by rahushen** [comment]
@mikewiebe... updated the PR with test changes and also added ip multicast NU piece.

**Comment by mikewiebe** [comment]
👍

**Pull request merged**